### ISSUE TITLE
ERROR: manifest for wodby/drupal-php:7.2-dev-4.12.3 not found

### ISSUE THREAD

**Issue by stjepan-ws** [wodby/docker4drupal#365]
***Codebase***
Built-in vanilla Drupal or mounted codebase

**Describe the bug**
tried to built most the project (docker-compose up) and errors with the title error

**Output of `docker info`**
```
Pulling php (wodby/drupal-php:7.2-dev-4.12.3)...
ERROR: manifest for wodby/drupal-php:7.2-dev-4.12.3 not found

```

**Contents of your `docker-compose.yml`**
```
it's the last build from your source, untouched
```

**Contents of your `.env`**
```
just the project title changed fomr your original
```

**Logs output `docker-compose logs`**
```
Paste here
```

**Comment by PavelPrischepa** [comment]
It [failed to build](https://travis-ci.org/wodby/drupal-php/jobs/503773252) so you may try previous version `wodby/drupal-php:7.2-dev-4.12.2`.

**Comment by csandanov** [comment]
There was the problem with the global composer package `hirak/prestissimo` as was reported at https://github.com/drupal-composer/drupal-project/issues/473 and https://github.com/drupal-composer/drupal-scaffold/issues/101. I don't know why it started to happen just now, probably something somewhere in the drupal composer project dependencies has changed. We've released `-4.12.4` with the fix.

Also, if you want a stable working environment, I recomend using the latest [stable release](https://github.com/wodby/docker4drupal/releases) tag or just download the archive from github releases instead of using master branch that [auto-updated by the both](https://github.com/wodby/docker4drupal#maintenance) and can be unstable.

**Comment by stjepan-ws** [comment]
Most usually I just copy/paste `.env` and `docker-compose.yml` for my projects from the master and not tagged stable releases, since it's just for local development.
Thank you for your support!
